packages/web/utils/ensHelpers.ts

```diff
@@ -0,0 +1,16 @@
+//import { CONFIG } from 'config';
+import { ethers } from 'ethers';
+//const { mainnetRPC } = CONFIG;
+
+//replace this with the mainnetRPC from above after testing and adding env, uncomment both mainnetRPC and CONFIG
+const mainnetProvider = new ethers.providers.JsonRpcProvider('https://ethereum.publicnode.com');
```

Comment:
If you search the project for other frontend variables, you should be able to figure out how to get this to work. You may need to add it to the Dockerfile and/or github action template

Reply:
If you look in `packages/web/config.ts`, it should be apparent how to specify a default value for a configuration value.